server/channels/app/post_metadata.go

```diff
@@ -858,7 +862,9 @@ func cacheLinkMetadata(requestURL string, timestamp int64, og *opengraph.OpenGra
 		Permalink: permalink,
 	}
 
-	platform.LinkCache().SetWithExpiry(strconv.FormatInt(model.GenerateLinkMetadataHash(requestURL, timestamp), 16), metadata, platform.LinkCacheDuration)
+	if err := platform.LinkCache().SetWithExpiry(strconv.FormatInt(model.GenerateLinkMetadataHash(requestURL, timestamp), 16), metadata, platform.LinkCacheDuration); err != nil {
+		mlog.Warn("Failed to cache link metadata", mlog.String("request_url", requestURL), mlog.Err(err))
```

Comment:
Not a blocker: Would you please add a `rctx request.CTX` as the first argument to `cacheLinkMetadata` and pass it along? Then we can use it to log the warning instead of using the global logger.

Reply:
Done!